example/advection/advection_package.cpp

```diff
@@ -276,6 +298,50 @@ void PostFill(MeshBlockData<Real> *rc) {
       });
 }
 
+// Example of how to enroll a history function.
+// Templating is *NOT* required and just implemented here to reuse this function
+// for testing of the UserHistoryOperations curently available in Parthenon (Sum, Min,
+// Max), which translate to the MPI reduction being called over all ranks. T should be
+// either Kokkos::Sum, Kokkos::Min, or Kokkos::Max.
+template <typename T>
+Real AdvectionHst(MeshData<Real> *md) {
+  auto hydro_pkg =
```

Comment:
This should probably say `advection_pkg`

Reply:
Yes, also it's not even used in this function :D So I removed it.